config/services/blik_payment/payum/factory.php

```diff
@@ -5,13 +5,14 @@
 namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
 use CommerceWeavers\SyliusTpayPlugin\BlikPayment\Payum\Factory\GatewayFactory;
-use Payum\Core\Bridge\Symfony\Builder\GatewayFactoryBuilder;
+use CommerceWeavers\SyliusTpayPlugin\Payum\Factory\TpayGatewayFactoryBuilder;
 
 return function(ContainerConfigurator $container): void {
     $services = $container->services();
 
-    $services->set('commerce_weavers_sylius_tpay.blik_payment.payum.factory.gateway', GatewayFactoryBuilder::class)
+    $services->set('commerce_weavers_sylius_tpay.blik_payment.payum.factory.gateway', TpayGatewayFactoryBuilder::class)
```

Comment:
Wouldn't parent definition work in this case to not redeclare this cache and class thing all the time?